.github/workflows/cdh_basic.yml

```
@@ -54,10 +54,14 @@ jobs:
         run: |
           sudo apt-get update && sudo apt-get install -y protobuf-compiler
 
+      - name: Install cryptsetup
+        run: |
+          sudo apt-get update && sudo apt-get install -y libcryptsetup-dev pkg-config
```

no need to update again:
```suggestion
        run: sudo apt install -y libcryptsetup-dev pkg-config
```

(ditto for other places)